src/firefly/js/templates/lightcurve/LcConverterFactory.js

```diff
@@ -60,15 +104,19 @@ export function getAllConverterIds() {
     return Object.keys(converters);
 }
 
-export function getConverter(datasetConverterId) {
-    const converter = datasetConverterId && converters[datasetConverterId];
+export function getConverter(converterId) {
+    const converter = converterId && converters[converterId];
     if (!converter) {
-        logError(`Unable to find dataset converter ${datasetConverterId}`);
+        logError(`Unable to find dataset converter ${converterId}`);
         return;
     }
     return converter;
 }
 
+export function getMissionName(converterId) {
+    const converterData = converterId && converters[converterId];
+    return get(converterData, 'missionName', converterId);
```

Comment:
I think missionName as it is is good, you might want to pass a mission name that has camel humps, like NExSci ;-)

BTW, a generic mission `unknown` mission name could be also called our `user defined` case - @tgoldina @robyww @cwang2016 DM-9491